### ISSUE TITLE
Release your changes for Swift 4

### ISSUE THREAD

**Issue by akabab** [KyoheiG3/SimpleAlert#28]
Could you please release your changes made in #27 to be available in a new patch version in cocoa pods ? thanks

**Comment by akabab** [comment]
[cocoapods] for those who temporary wants to apply the changes, you must set in your Podfile
```
pod 'SimpleAlert', :git => 'https://github.com/KyoheiG3/SimpleAlert.git'
```

**Comment by KyoheiG3** [comment]
Released in [2.1.0](https://github.com/KyoheiG3/SimpleAlert/releases/tag/2.1.0). Thanks.

**Comment by akabab** [comment]
perfect!
